src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -29,18 +29,19 @@
     @RequestMapping("/")
     public String index(Model model) {
         Item item = new Item(0, "item");
-        item.setDescription("Des");
-        item.setLocation("locale");
-        item.setAssignedTo(1);
-        item.setStatusId(0);
+        //item.setDescription("Des");
```

Comment:
You should remove this rather than commenting it. If we need it again we can use version history to retrieve it

Reply:
Thank you.  Yes.  Correct.
Commented out code is a no-no in version control.
1) You can go back in history to see what used to be there.
2) If the work is experimental in nature, you can put it in a branch.